doc/sphinx/practical-tools/utilities.rst

```diff
@@ -659,6 +659,38 @@ See the man page of ``coqdep`` for more details and options.
 Both Dune and ``coq_makefile`` use ``coqdep`` to compute the
 dependencies among the files part of a Coq project.
 
+.. _coqnative:
+
+Split compilation of native computation files
+---------------------------------------------
+
+Coq features a :tacn:`native_compute` tactic to provide fast computation in the
+kernel. This process performs compilation of Coq terms to OCaml programs using
+the OCaml compiler, which may cause an important overhead. Hence native
+compilation is an opt-in configure flag.
+
+When native compilation is activated, Coq generates the compiled files upfront,
+i.e. during the ``coqc`` invocation on the corresponding ``.v`` file. This is
+impractical because it means one must chose in advance whether they will use
+a native-capable Coq installation. In particular, activating native compilation
+forces the recompilation of the whole Coq installation. See
+:ref:`command line options <command-line-options>` for more details.
+
+Starting from Coq 8.14, a new binary ``coqnative`` is available. It allows
+performing split native compilation by generating the native compute files out
+of the compiled ``.vo`` file rather than out of the source ``.v`` file.
+
+The ``coqnative`` command takes a name *file.vo* as argument and tries to
+perform native compilation on it. It assumes that the Coq libraries on which
+*file.vo* depends have been first compiled to their native files, and will fail
+otherwise. It accepts the ``-R``, ``-Q``, ``-I`` and ``-nI`` arguments with the
```

Comment:
I don't recall `-nI` being documented anywhere (I was not even aware of its existence until recently). Anyway, if this one too is a no-op when passed to `coqnative`, that should probably be made explicit.

Reply:
`-nI` is really internal so that is the reason it was not documented.

Reply:
Should I remove it from the doc?

Reply:
Seems a good idea, as it is likely we can get rid of this option for 8.14